### ISSUE TITLE
as the file is read through fuse, the cache ratio of the alluxio cluster is getting lower and lower

### ISSUE THREAD

**Issue by wwq2333** [Alluxio/alluxio#17703]
**Alluxio Version:**
2.x master [(commit)](https://github.com/Alluxio/alluxio/commit/ea7ac65e96b40a5287a3caf900197e998a99f40e)

**Describe the bug**
we use fuse to access a dataset of about 2.6TB; 
before starting the training, the cache has been preheated with `fs load` to ensure that the cache ratio is 100%; and there is still some remaining cache space in the cluster;
![image](https://github.com/Alluxio/alluxio/assets/12540485/1633827b-6872-4955-be8f-946e2815d250)
![image](https://github.com/Alluxio/alluxio/assets/12540485/ec0958da-f575-45e1-9006-30e37f83196f)
![image](https://github.com/Alluxio/alluxio/assets/12540485/51e0e66e-4762-44cc-917e-8f8136bd218c)

as the training progresses, the cache ratio seen through `fs ls` is gradually decreasing, and the cache space of the cluster is gradually increasing; alluxio worker requests ufs traffic bandwidth is getting higher and higher;

![image](https://github.com/Alluxio/alluxio/assets/12540485/c3813b02-3d32-4519-88fd-69314b3ab826)
![image](https://github.com/Alluxio/alluxio/assets/12540485/d82544d3-2b2f-43fe-b5e6-9fe92ce2bc25)

and we try to use `fs load` to warm up again, but the amount of preheated data far exceeds the total amount of the dataset;

![image](https://github.com/Alluxio/alluxio/assets/12540485/b3846e60-e9be-4b5e-9d0f-2f705f0623eb)

**Expected behavior**
the cache ratio can be stabilized at 100%, and access to UFS should be as rare as possible;

**Additional context**

- 1 master and 14 worker and 25+ fuse
  - alluxio.user.file.replication.max=1 
  - alluxio.worker.tieredstore.level0.watermark.high.ratio=1 
  - alluxio.worker.tieredstore.level0.watermark.low.ratio=1
- this dataset has 110 files and total size is 2632.25 GB;
- ~in multiple fuse logs, there are requests to workers that fail, and then retry other workers (very frequently);~
  - ~https://github.com/Alluxio/alluxio/issues/16782~
  - ~this behavior maybe cause the same cached data block to exist on multiple workers, and with accumulation, the worker will eventually store the upper limit and begin to eliminate other data ?~
  - we switch worker to containerNetwork and fuse read fail no longer appears, but the cache ratio continues to decline;

the test cluster is newly started, and the log accumulation is relatively small, but the phenomenon is the same:
- [fuse.log](https://github.com/Alluxio/alluxio/files/11913474/fuse.log)
- [worker-0.log](https://github.com/Alluxio/alluxio/files/11913520/worker-0.log)
- other worker logs are similar, is there a debug flag that can print out more detailed information?

**Comment by jja725** [comment]
@wwq2333 hey, do you mind pasting more worker/fuse logs here so we can have a better understanding of why we evict the blocks when reading the file?

**Comment by ssz1997** [comment]
@wwq2333 Maybe double check the value of `alluxio.user.file.passive.cache.enabled`? I see Fluid turned it off by default, but just in case.

**Comment by LuQQiu** [comment]
Thanks @wwq2333!
An assumption is that the data is replicated across the cluster, while data is replicating, existing data is also evicted. 
```
  public static final PropertyKey USER_FILE_REPLICATION_MAX =
      intBuilder(Name.USER_FILE_REPLICATION_MAX)
          .setDefaultValue(-1)
          .setDescription("The target max replication level of a file in Alluxio space. Setting "
              + "this property to a negative value means no upper limit.")
          .setScope(Scope.CLIENT)
          .build();
  public static final PropertyKey USER_FILE_REPLICATION_MIN =
      intBuilder(Name.USER_FILE_REPLICATION_MIN)
          .setDefaultValue(0)
          .setDescription("The target min replication level of a file in Alluxio space.")
          .setScope(Scope.CLIENT)
          .build();
```
would it be helpful to set both of them to one?

**Comment by LuQQiu** [comment]
@wwq2333 thanks for the issue, the behavior is quite strange.
Can you try set both of the following properties to be 1?
```
  public static final PropertyKey USER_FILE_REPLICATION_MAX =
      intBuilder(Name.USER_FILE_REPLICATION_MAX)
          .setDefaultValue(-1)
          .setDescription("The target max replication level of a file in Alluxio space. Setting "
              + "this property to a negative value means no upper limit.")
          .setScope(Scope.CLIENT)
          .build();
  public static final PropertyKey USER_FILE_REPLICATION_MIN =
      intBuilder(Name.USER_FILE_REPLICATION_MIN)
          .setDefaultValue(0)
          .setDescription("The target min replication level of a file in Alluxio space.")
          .setScope(Scope.CLIENT)
          .build();
```
@ssz1997 thanks for validating one of the assumption

**Comment by jja725** [comment]
yeah, they are duplicate, we are caching the new blocks during reading and evicting the old blocks, which cause the caching ratio of some files to drop. In the short term, we have to increase the storage size since we don't support the policy to send load requests to the worker with enough storage(we just send the new task to whatever worker finishes the task earlier).

**Comment by Haoning-Sun** [comment]
@jja725 Is it possible to use the following configuration to keep the worker from caching new data, thereby maintaining the existing cached data.  Will there be other effects?
```alluxio.user.file.readtype.default=NO_CACHE```

**Comment by jja725** [comment]
NO_CACHE seems like a good idea as well @Haoning-Sun @wwq2333

**Comment by jja725** [comment]
Discussed with the customer offline, the error message is misleading and the read content is correct. The user switches to use NO_CACHE read type and no error message is seen. So I would close the ticket for now.
